File: week18/p_12906.js
#javascript

function solution(arr)
{
    let answer = [];
    
    
    for (let i = 0; i < arr.length; i++){
        answer.length===0? answer.push(arr[i]): (arr[i]!==answer[answer.length-1]?answer.push(arr[i]):null)
       
    }
    return answer
    
}

File: week18/p_12906.py


arr = [1]
def solution(arr):
    ans = []
    print(len(ans),arr[0])
    for i in range(len(arr)):
        if len(ans)== 0:
            ans.append(arr[i])
        else:
            if arr[i] == ans[-1]:
                continue
            else:
                ans.append(arr[i])
    return ans

print(solution(arr))
